### DISCUSSION TITLE
Add wide range of emoji instead of 'like' button

### DISCUSSION THREAD

**Discussion by brootle** [deso-protocol/dips#215] [Medium CIPs]
I suggest to add pop up window to show when clicking 'like' button and show a set of different emojis. This is cool when people can select different  emoji  as their reaction to content of other users and this would increase user interaction. Also instead of making a comment with emoji people will be able just to click emoji, so this will potentially save a lot of space as it will reduce number of comments.

PS at least add crypto-related emoji like 🚀🔥🤑
![image](https://user-images.githubusercontent.com/20090030/153426754-538d4b5d-ec72-4cc3-bcac-0f904aa836bd.png)

**Comment by oO** [comment]
If we're ok with a user only being able to vote for a single emoji, this could be implemented pretty simply by allowing the emoji's unicode to be stored as extradata on the LIKE transaction. So you could have posts with multiple emoji's just like Slack and Discord, but the difference would be that each user could only have a single LIKE, not multiple ones.

One interesting side-effect is that this can be used as a polling system ( just like Discord does, but without the double-voting )

❤️23 🎂12 🚀3
